node/src/chain_spec.rs

```diff
@@ -445,7 +447,21 @@ fn devnet_testnet_genesis(
 		treasury: Default::default(),
 
 		// EVM compatibility
-		evm: EVMConfig { accounts: { BTreeMap::new() } },
+		evm: EVMConfig {
+			accounts: {
+				let mut map = BTreeMap::new();
+				map.insert(
+					H160::from_str("0xfFFFffFF000000000000000000000000000007e2").expect("invalid address"),
+					fp_evm::GenesisAccount {
+					 	nonce: Default::default(),
+						balance: Default::default(),
+						storage: Default::default(),
+						code: vec![0x60, 0x00, 0x60, 0x00, 0xfd]
+					}
+				);
+				map
```

Comment:
This code is not necessary. The precompile is "defined" in `runtime/devnet/precompiles.rs`
```suggestion
		evm: EVMConfig { accounts: { BTreeMap::new() } },
```